mlir/include/mlir/IR/PDLPatternMatch.h.inc

```diff
@@ -0,0 +1,992 @@
+//===- PDLPatternMatch.h - PDLPatternMatcher classes -------==---*- C++ -*-===//
```

Comment:
Not clear to me why is this a `.h.inc` extension? Seems like a regular `.h` to me.

Isn't it standalone? If so can you point out why?

Reply:
ping?

Reply:
Its not as it depends on data structures defined above it, but also it doesn't make sense to use in isolation as the functionality is not self-contained.